README.md

```diff
@@ -86,3 +86,7 @@ The docs are generated with every build and the changes should be committed when
 
 The plugin is still under development so the format could change.
 For more info see the plugin repository on GitHub.
+
+## Allure reports
```

Comment:
Use block of code instead of just highlight

Reply:
done